hack/build-antrea-windows-all.sh

```diff
@@ -78,12 +78,25 @@ if $PULL; then
     ARGS="$ARGS --pull"
 fi
 
+GO_VERSION=$(head -n 1 build/images/deps/go-version)
+
+if [[ "${DOCKER_REGISTRY}" == "" ]]; then
+    docker pull --platform linux/amd64 golang:$GO_VERSION
+else
+    docker pull --platform linux/amd64 ${DOCKER_REGISTRY}/antrea/golang:$GO_VERSION
+    docker tag ${DOCKER_REGISTRY}/antrea/golang:$GO_VERSION $GO_VERSION
+fi
```

Comment:
maybe move this right before line 102 below, as I assume this is specifically for the last build step?

Reply:
Sure, updated.